### ISSUE TITLE
Unable to intercept Magento\Framework\App\ErrorHandler or use different error_handler

### ISSUE THREAD

**Issue by stefandoorn** [magento/magento2#3387]
The way the ErrorHandler is loaded in Framework\App\Bootstrap probably prevents interception on the ErrorHandler class. Reason I would like to use this, is to adjust the ErrorHandler in a more developer friendly version (Whoops).

It makes sense, as the error handler needs to be registered before the object manager will be loaded. But, it would be great if there is a method to register another error_handler after that is done.

So basically it would be:
- Register default Magento error handler
- Load object manager
- Register default Magento error handler again using object manager

The second moment would allow for interception I believe. Or, am I overseeing a method to register a different error_handler?

**Comment by Vinai** [comment]
How about simply using [`set_error_handler`](http://php.net/manual/en/function.set-error-handler.php)?

**Comment by stefandoorn** [comment]
Can do that, but that's quite late in the process then. Let's say I want the error from an erroneus di.xml also in my custom error handler, how would that work?

**Comment by piotrekkaminski** [comment]
Thank you for your submission.

We recently made some changes to the way we process GitHub submissions to more quickly identify and respond to core code issues.

Feature Requests and Improvements should now be submitted to the new [Magento 2 Feature Requests and Improvements](https://community.magento.com/t5/Magento-2-Feature-Requests-and/idb-p/feature-requests) forum (see details [here](https://community.magento.com/t5/News-Announcements/Improvements-to-GitHub-Management/m-p/44572#M96)).

We are closing this GitHub ticket and have moved your request to the new [forum](https://community.magento.com/t5/Magento-2-Feature-Requests-and/Unable-to-intercept-Magento-Framework-App-ErrorHandler-or-use/idi-p/45324).
